tests/devices/test_default_qubit_autograd.py

```diff
@@ -541,3 +543,120 @@ def test_partial_subsystem(self, mocker):
 
         assert np.all(res == state)
         spy.assert_called()
+
+
+@pytest.mark.autograd
+class TestObservableWithObjectReturnType:

```

Comment:
Likely this test case would better be fitted in the `tests/interfaces/test_autograd_qnode.py` file or `tests/test_qubit_device.py`. The feature is not specific to `default.qubit.autograd` (this test file is though).

Reply:
fixed